### ISSUE TITLE
vp8_decoder: the flush handler should not discard the codec config, only the stored stream data

### ISSUE THREAD

**Issue by juanrubio** [tizonia/tizonia-openmax-il#354]
The flush handler should not do a full reset of the stream parameters. It should only discard any stream data found in the internal buffers, but keep the stream parameter information:

```
static OMX_ERRORTYPE
vp8d_prc_port_flush (const void * ap_obj, OMX_U32 a_pid)
{
  vp8d_prc_t * p_prc = (vp8d_prc_t *) ap_obj;
  if (OMX_ALL == a_pid || ARATELIA_VP8_DECODER_INPUT_PORT_INDEX == a_pid)
    {
      release_input_header (p_prc);
      reset_stream_parameters (p_prc);
    }
  if (OMX_ALL == a_pid || ARATELIA_VP8_DECODER_OUTPUT_PORT_INDEX == a_pid)
    {
      release_output_header (p_prc);
    }
  return OMX_ErrorNone;
}
```

**Comment by juanrubio** [comment]
Committed a potential fix here 9de14f32aad34ebd1b9429be3bc4f13645941711. 

I could not find any libvpx API to flush the decoder context, so this fix might be all that needs to be done.

@CapOM, could you please re-check your test case with this fix in? Thanks!

**Comment by juanrubio** [comment]
I believe this is working now. So I'm closing this issue.
